common/dtd/ldmlSupplemental.dtd

```diff
@@ -12,10 +12,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST version number CDATA #REQUIRED >
     <!--@MATCH:any-->
     <!--@METADATA-->
-<!ATTLIST version cldrVersion CDATA #FIXED "42" >
+<!ATTLIST version cldrVersion CDATA #FIXED "43" >
     <!--@MATCH:version-->
     <!--@VALUE-->
-<!ATTLIST version unicodeVersion CDATA #FIXED "14.0.0" >
+<!ATTLIST version unicodeVersion CDATA #FIXED "15.0.0" >
```

Comment:
Hmm. CLDR 42 went out with the wrong `unicodeVersion`. Not sure how much of an issue that is. Will add an agenda item to discuss whether we should update this on the maint-42 branch.